3MF Core Specification.md

```diff
@@ -308,12 +308,12 @@ The language of the contents of a 3MF Document (typically useful for content pro
 
 # Chapter 3. 3D Models
 
-The _model_, in this specification, refers to the object or objects to be created via 3D manufacturing processes as a single operation. It might include a single object, multiple homogenous objects, multiple heterogeneous objects, an object fully enclosed in another object, or multiple objects in an interlocked and inseparable _assembly_.
+The _model_, in this specification, refers to the object or objects to be ultimately created via 3D manufacturing processes as a single operation. It might include a single object, multiple homogenous objects, multiple heterogeneous objects, an object fully enclosed in another object, or multiple objects in an interlocked and inseparable _assembly_. A 3MF Document can be in an intermediate state in which none of the defined objects is yet ready to be manufactured, in which case no actual output object is expected.

```

Comment:
I don't think the "ultimately" is needed here, we are not changing the core intention of 3MF to be a manufacturing format so the end state of being created by 3D manufacturing is unchanged and since the intermediate state (from editors, simulators etc.) is a possibility but not a certainty we don't necessarily need to make the distinction.